core/src/main/kotlin/org/evomaster/core/problem/rest/service/AbstractRestSampler.kt

```diff
@@ -245,6 +245,38 @@ abstract class AbstractRestSampler : HttpWsSampler<RestIndividual>() {
         initAdHocInitialIndividuals()
     }
 
+    private fun getEndPointsToSkipSimilarHelper(swagger: OpenAPI)
+    :List<String>{
+
+        // all endpoints
+        val all = swagger.paths.map{it.key}
+
+        if(all.none { it == configuration.endpointFocus }){
+            throw IllegalArgumentException(
+                "Invalid endpointFocus: ${configuration.endpointFocus}. " +
+                        "\nAvailable:\n${all.joinToString("\n")}")
+        }
+
+        // filter all items which do not contain the user provided item
+        return all.filterNot { it.toString().contains(configuration.endpointFocus.toString()) }
+    }
+
+    private fun getEndPointsToSkipExactHelper(swagger: OpenAPI)
+            :List<String>
+    {
+        val all = swagger.paths.map{it.key}
+
+        if(all.none { it == configuration.endpointFocus }){
+            throw IllegalArgumentException(
+                "Invalid endpointFocus: ${configuration.endpointFocus}. " +
+                        "\nAvailable:\n${all.joinToString("\n")}")
+        }
+
+        return all.filter { it != configuration.endpointFocus }
```

Comment:
these 2 functions have the same code, with just last line that is different. should avoid code duplication